pkg/puppetdb/nodes_test.go

```diff
@@ -10,14 +10,14 @@ func TestTasks(t *testing.T) {
 
 	// Test without query
 	setupGetResponder(t, "/pdb/query/v4/nodes", "", "nodes-response.json")
-	actual, err := pdbClient.Nodes("")
+	actual, err := pdbClient.Nodes("", Pagination{})
```

Comment:
Mentioned making the 2nd arg a pointer so this can become `pdbClient.Nodes("", nil)`.